### DISCUSSION TITLE
tput: unknown terminal "xterm-ghostty"

### DISCUSSION THREAD

**Discussion by ztomer** [ghostty-org/ghostty#3785] [Q&A] [answered]
tl;dr: Calling tput returns:
```
ztomer@raspberrypi:/opt/pihole$ tput
tput: unknown terminal "xterm-ghostty"
```
This is obviously a tput problem. Is there a way to report a custom terminal identifier in Ghostty?

Thanks!

Details:

I got Pi-Hole installed on an RPI 4, with an Ubuntu 24.04 OS.
I'm using this the MacOS version of Ghostty, with an SSH session to the RPI.

When I run `pihole -up -d`, I get the following error:

<img width="1064" alt="image" src="https://github.com/user-attachments/assets/9e2143bb-d4a8-4551-a8fa-715eaf7c4613" />

`pihole` is a bash script that in this instance calls `/opt/pihole/update.sh`. 

Following up on this, COL_TABLE is a file in `/opt/pihole/COL_TABLE`
and line 2 is `if ([ -t 1 ] && [ $(tput colors) -ge 8 ]) || [ "${WEBCALL}" ]; then`

Calling tput directly nets:
```
ztomer@raspberrypi:/opt/pihole$ tput
tput: unknown terminal "xterm-ghostty"
```

**Comment by jcollie** [comment, marked as answer]
https://ghostty.org/docs/help/terminfo

**Comment by AjayShanker-geek** [comment]
Quick workaround is to use `export TERM=xterm-256color` in my bashrc
